### PR TITLE
chore: migrate hosting category from inquirer to prompter library

### PR DESCRIPTION
**Pull request by lazpavel**
#### Description of changes
- migrate hosting category from inquirer to prompter library

#### Checklist
- [x] PR description included
- [x] `yarn test` passes 
- [x] Tests are [changed or added](https://github.com/aws-amplify/amplify-cli/blob/dev/CONTRIBUTING.md#tests)

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

**Review by sobolk — APPROVED** [review]
Good to merge with passing tests.

File: packages/amplify-category-hosting/lib/S3AndCloudFront/helpers/configure-CloudFront.js [hunk 1]
```diff
@@ -75,46 +65,12 @@ async function configure(context) {
...
-        default: true,
-      },
-    ];
-
-    const answers = await inquirer.prompt(questions);
-    DistributionConfig.DefaultRootObject = answers.DefaultRootObject;
-    DistributionConfig.DefaultCacheBehavior.DefaultTTL = answers.DefaultCacheDefaultTTL;
-    DistributionConfig.DefaultCacheBehavior.MaxTTL = answers.DefaultCacheMaxTTL;
-    DistributionConfig.DefaultCacheBehavior.MinTTL = answers.DefaultCacheMinTTL;
+    DistributionConfig.DefaultRootObject = await prompter.input('default object to return from origin', { initial: DistributionConfig.DefaultRootObject });
```
Comment (evcodes) [inline packages/amplify-category-hosting/lib/S3AndCloudFront/helpers/configure-CloudFront.js:68]: nit: lowercase d here for default where the other messages have a uppercase D

**Review by evcodes — DISMISSED** [review]
one tiny nit but overall lgtm!! Great work

File: packages/amplify-category-hosting/lib/S3AndCloudFront/helpers/configure-CloudFront.js (hunk 1 above)
Comment (lazpavel) [reply packages/amplify-category-hosting/lib/S3AndCloudFront/helpers/configure-CloudFront.js:68]: agree that isn't standard, but for the scope of this PR will leave as is as this it's how it's been in the previous version: https://github.com/aws-amplify/amplify-cli/pull/11679/files#diff-09e35a12cba2a09eb33793e5438207fb56462e5359e9a4042f5fd8c513cf41ccL82, a change here will probably break some prompter tests

**Review by evcodes — APPROVED** [review]

**Review by evcodes — APPROVED** [review]
approved pending e2e tests 🙂

**Pull request merged**